meshtastic/tests/test_analysis_coverage.py

```
@@ -0,0 +1,452 @@
+        }
+    )
+    feather.write_feather(slog_data, str(slog_dir / "slog.feather"))
+
+    with pytest.raises((FileNotFoundError, OSError)):
+        create_dash(str(slog_dir))
+
+
+@pytest.mark.unit
+def test_main_uses_default_slog_path(
+    monkeypatch: pytest.MonkeyPatch,
+    cli_exit_capture: dict[str, Any],
```

Ruff (0.15.7)

[warning] Unused function argument: `cli_exit_capture`

(ARG001)